<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>company_orders_search</class>
 <widget class="QDialog" name="company_orders_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_title_Report">
     <property name="minimumSize">
      <size>
       <width>411</width>
       <height>84</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>411</width>
       <height>84</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ImgReport">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Recource.qrc">:/rec/img/report.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbReport">
        <property name="minimumSize">
         <size>
          <width>184</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Отчет Заказов Компании</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTableWidget" name="twOrg"/>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>451</width>
       <height>291</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>451</width>
       <height>291</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="cbComp_name">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>10</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbID_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>116</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Название Компании </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReport">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Получить Данные</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>43</y>
        <width>431</width>
        <height>241</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="leDate"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leOrder_number"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ФИО Клиента </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Сумма Закакза </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leSum_order"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leFio_client"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbName_2">
         <property name="text">
          <string>Номер заказа </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbINN_2">
         <property name="text">
          <string>Дата Заказа</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>321</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>321</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад </string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="teResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QFrame" name="com_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>191</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbID_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>116</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>название Компании</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leComp_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>193</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Recource.qrc"/>
 </resources>
 <connections/>
</ui>
